packages/vscode-extension/src/webview/components/DeviceSettingsDropdown.tsx

```diff
@@ -112,6 +113,26 @@ function DeviceSettingsDropdown({ children, disabled }: DeviceSettingsDropdownPr
             <span className="codicon codicon-location" />
             Set Device Location
           </DropdownMenu.Item>
+          <Label>Device Frame</Label>
+          <form>
+            <div style={{ display: "flex", alignItems: "center" }}>
```

Comment:
@jakub-gonet https://github.com/software-mansion/react-native-ide/pull/422#pullrequestreview-2168429727 
`change the color of switch to match the rest of the theme.` - What do you exactly mean? Do you mean color the checked state color?

Reply:
Yes, exactly that.

Reply:
@jakub-gonet `--navy-light-60` is fine?

Reply:
@jakub-gonet `we need to fix weird layout issue that happens after switching` - if you think about the notch area, it happens because of the way how the mask looks, we can disable mask or change it

Reply:
@jakub-gonet now it looks a bit better and the unexpected width increase does not happen anymore

https://github.com/user-attachments/assets/2a3cb9d1-84ca-40a0-b6d1-97f986b05e5d